com.github.expresslrs.ExpressLRSConfigurator.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.github.expresslrs.ExpressLRSConfigurator
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: expresslrs-configurator
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --socket=system-bus
+  - --filesystem=/tmp
+  - --filesystem=/run/udev:ro
+  - --filesystem=~/.platformio
+modules:
+  - name: git
+    make-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    make-install-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    post-install:
+      - find . -type f -name perllocal.pod -delete
+    sources:
+      - type: archive
+        sha256: 814641d7f61659cfbc17825d0462499ca1403e39ff53d76a8512050e6483e87a
+        url: https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.3.tar.xz
+        x-checker-data:
+          type: html
+          url: https://mirrors.edge.kernel.org/pub/software/scm/git/
+          version-pattern: git-([\d\.-]+).tar.xz
+          url-template: https://mirrors.edge.kernel.org/pub/software/scm/git/git-$version.tar.xz
+  - name: expresslrs-configurator
+    buildsystem: simple
+    build-commands:
+      - ar x expresslrs-configurator_*.deb
+      - rm expresslrs-configurator_*.deb
+      - tar xf data.tar.xz
+      - cp -r 'opt/ExpressLRS Configurator' '/app/ExpressLRS-Configurator'
+      - mkdir -p /app/share/icons/hicolor
+      - chmod -R a-s,go+rX,go-w "/app/ExpressLRS-Configurator"
+      - install expresslrs-configurator.sh /app/bin/expresslrs-configurator
+      - install -Dm644 expresslrs-configurator.png /app/share/icons/hicolor/scalable/apps/com.github.expresslrs.ExpressLRSConfigurator.png
```

Comment:
a PNG is NOT SCALABLE.

Reply:
Thx, changed to 512x512

Reply:
you should use the icons that are in the debian package too.

Reply:
Done, would there be an easy way to copy all or would I need to just have a line for every icon size?

Reply:
There's no need to copy more than 512x512 unless it's used by the app directly.